test/python/test_nexus.py

```diff
@@ -236,10 +144,11 @@ async def test_nexus_2_mirror_kill_one(containers, create_nexus):
 
 
 @pytest.mark.asyncio
+@pytest.mark.skip
```

Comment:
Is there a reason we are skipping this now?

Reply:
Yeah, the test cant run at the same time/run you need to pick one :) Until we run these things automated it's like, oke, take your pick. working on the automation part.